Resources/doc/reference/action_export.rst

```diff
@@ -12,20 +12,94 @@ This document will cover the Export action and related configuration options.
 Basic configuration
 -------------------
 
+If you have registered the ``SonataExporterBundle`` bundle in the kernel of your application,
+you can benefit from a lot of flexibility:
+
+* You can configure default exporters globally.
+* You can add custom exporters, also globally.
+* You can configure every default writer.
+
+See `the exporter bundle documentation`_ for more information.
```

Comment:
I mean https://github.com/sonata-project/SonataAdminBundle/pull/4029 . I think you should just remove this § for now, I'll add it back when and if my PR gets merged one day.